src/usr/local/www/status_openvpn.php

```diff
@@ -143,130 +142,109 @@ function(index,row) { jQuery(row).fadeOut(1000); }
-	<tr>
-		<td colspan="6" class="listtopic">
-			<?=htmlspecialchars($server['name']);?> <?=gettext("Client connections"); ?>
-		</td>
-	</tr>
-	<tr>
-		<td>
-			<table style="padding-top:0px; padding-bottom:0px; padding-left:0px; padding-right:0px" class="tabcont sortable" width="100%" border="0" cellpadding="0" cellspacing="0" summary="connections">
-			<tr>
-				<td class="listhdrr"><?=gettext("Common Name"); ?></td>
-				<td class="listhdrr"><?=gettext("Real Address"); ?></td>
-				<td class="listhdrr"><?=gettext("Virtual Address"); ?></td>
-				<td class="listhdrr"><?=gettext("Connected Since"); ?></td>
-				<td class="listhdrr"><?=gettext("Bytes Sent"); ?></td>
-				<td class="listhdrr"><?=gettext("Bytes Received"); ?></td>
-			</tr>
+<div class="panel panel-default">
+		<div class="panel-heading"><h2 class="panel-title"><?=htmlspecialchars($server['name']);?> <?=gettext('Client connections')?></h2></div>
+		<div class="panel-body table-responsive">
+			<table class="table table-striped table-hover">
+				<thead>
+					<tr>
+						<th><?=gettext("Common Name")?></th>
+						<th><?=gettext("Real Address")?></th>
+						<th><?=gettext("Virtual Address"); ?></th>
+						<th><?=gettext("Connected Since"); ?></th>
+						<th><?=gettext("Bytes Sent")?></th>
+						<th><?=gettext("Bytes Received")?></th>
+					</tr>
+				</thead>
+				<tbody>
+			
 <?php
 		foreach ($server['conns'] as $conn):
 ?>
-			<tr id="<?php echo "r:{$server['mgmt']}:{$conn['remote_host']}"; ?>">
-				<td class="listlr">
-					<?=$conn['common_name'];?>
-				</td>
-				<td class="listr">
-					<?=$conn['remote_host'];?>
-				</td>
-				<td class="listr">
-					<?=$conn['virtual_addr'];?>
-				</td>
-				<td class="listr">
-					<?=$conn['connect_time'];?>
-				</td>
-				<td class="listr">
-					<?=format_bytes($conn['bytes_sent']);?>
-				</td>
-				<td class="listr">
-					<?=format_bytes($conn['bytes_recv']);?>
-				</td>
-				<td class="list">
-					<img src="/themes/<?php echo $g['theme']; ?>/images/icons/icon_x.gif" height="17" width="17" border="0"
-					   onclick="killClient('<?php echo $server['mgmt']; ?>', '<?php echo $conn['remote_host']; ?>');" style="cursor:pointer;"
-					   id="<?php echo "i:{$server['mgmt']}:{$conn['remote_host']}"; ?>"
-					   title="<?php echo gettext("Kill client connection from") . " " . $conn['remote_host']; ?>" alt="delete" />
-				</td>
-			</tr>
+					<tr id="<?php echo "r:{$server['mgmt']}:{$conn['remote_host']}"; ?>">
+						<td><?=$conn['common_name'];?></td>
+						<td><?=$conn['remote_host'];?></td>
+						<td><?=$conn['virtual_addr'];?></td>
+						<td><?=$conn['connect_time'];?></td>
+						<td><?=format_bytes($conn['bytes_sent']);?></td>
+						<td><?=format_bytes($conn['bytes_recv']);?></td>
+						<td>
+							<img src="/themes/<?php echo $g['theme']; ?>/images/icons/icon_x.gif"
+							   onclick="killClient('<?php echo $server['mgmt']; ?>', '<?php echo $conn['remote_host']; ?>');" style="cursor:pointer;"
```

Comment:
We are employing the icon set found at www/bootstrap/glyphicons and deprecating the old images/icons/icon_* icons. It would be better to choose one of those glyphicons. They are typically used like so:

<i class="icon-large icon-question-sign"></i>